src/utils/githubEventHelpers.ts

```diff
@@ -105,7 +105,7 @@ export async function modifyProjectIssueField(
     octokit
   );
   const singleSelectOptionId = projectFieldNodeIDMapping[projectFieldOption];
-  const addIssueToGlobalIssuesProjectMutation = `mutation {
+  const modifyProjectIssueFieldMutation = `mutation {
```

Comment:
Fixing a bad name?

Reply:
yessir, this function definitely has nothing to do with adding issues to global issues project